wait-for

```diff
@@ -68,11 +68,11 @@ wait_for() {
 
   while :; do
     case "$PROTOCOL" in
-      tcp) 
+      tcp)
         nc -w 1 -z "$HOST" "$PORT" > /dev/null 2>&1
         ;;
       http)
-        wget --timeout=1 -q "$HOST" -O /dev/null > /dev/null 2>&1 
+        wget --timeout=1 --tries=1 -q "$HOST" -O /dev/null > /dev/null 2>&1
```

Comment:
Not sure if this is an issue for other platforms/environments.

Reply:
I'm running into an endless test run (at step `9 wget timeout does not double`) without the explicit `--tries` option. As far as I understood this is not a Docker specific issue per se, it's only made visible by the Docker Alpine test environment. I'm going to create a dedicated pull request for this one.

Reply:
-> #100

I would expect the checks in this pull request to fail and only succeed after #100 being merged.